dashboard/config/scripts_json/csp7-2022.script_json

```diff
@@ -0,0 +1,4216 @@
+          }
+        ]
+      },
+      "seeding_key": {
+        "activity_section.key": "a41423e8-6eeb-4cbc-8963-ddf16a8558cc",
+        "lesson_activity.key": "7dcb758d-b6b6-4afa-8df5-21d4792d616b"
+      }
+    },
+    {
+      "key": "02e0c6ec-aaca-4dde-a829-c7c5160a6960",
+      "position": 1,
+      "properties": {
+        "description": "Review the answers to the assessment with the class. Discuss any questions that come up and take note of topics where students may need extra review."
+      },
+      "seeding_key": {
+        "activity_section.key": "02e0c6ec-aaca-4dde-a829-c7c5160a6960",
+        "lesson_activity.key": "9eb8cdd5-bc38-4b63-bc81-9b0434f33c39"
+      }
+    },
+    {
+      "key": "029edb0b-d410-49af-ba43-6c92f094a9a3",
+      "position": 1,
+      "properties": {
+        "progression_name": "Unit 7 Exam"
+      },
+      "seeding_key": {
+        "activity_section.key": "029edb0b-d410-49af-ba43-6c92f094a9a3",
+        "lesson_activity.key": "98956100-80d5-4857-82bd-980452f92b52"
+      }
+    }
+  ],
+  "script_levels": [
+    {
+      "chapter": 1,
+      "position": 1,
+      "activity_section_position": 1,
+      "assessment": true,
+      "properties": {
+        "progression": "Check For Understanding"
+      },
+      "bonus": false,
+      "seeding_key": {
+        "script_level.level_keys": [
+          "U7L1 CFU Return2022"
+        ],
+        "lesson.key": "Parameters and Return Explore",
+        "lesson_group.key": "csp_parameters_return_values",
+        "script.name": "csp7-2022",
+        "activity_section.key": "9419b437-62ab-4fe9-a532-789c2d683277"
+      },
+      "level_keys": [
+        "U7L1 CFU Return2022"
+      ]
+    },
+    {
+      "chapter": 2,
+      "position": 1,
+      "activity_section_position": 1,
+      "assessment": false,
+      "properties": {
+        "progression": "App Investigation"
+      },
+      "bonus": false,
+      "seeding_key": {
+        "script_level.level_keys": [
+          "U7 Param Return Investigate Rock2022"
+        ],
+        "lesson.key": "Parameters and Return Investigate",
+        "lesson_group.key": "csp_parameters_return_values",
+        "script.name": "csp7-2022",
+        "activity_section.key": "055e1b7e-67de-4845-b95d-72284f2018c3"
+      },
+      "level_keys": [
+        "U7 Param Return Investigate Rock2022"
+      ]
+    },
+    {
+      "chapter": 3,
+      "position": 2,
+      "activity_section_position": 2,
+      "assessment": false,
+      "properties": {
+        "progression": "App Investigation"
+      },
+      "bonus": false,
+      "seeding_key": {
+        "script_level.level_keys": [
+          "U7 Param Investigate Poetry App2022"
```

Comment:
it looks like these levels exist already -- did they get created by a previous attempt to do the clone?

Reply:
Where do you see that level? I don't see it in prod:  `SELECT * FROM dashboard_production.levels WHERE (name LIKE '%CSP U7%');` ... unless that's not the right table

Reply:
Just searched levelbuilder with `SELECT name FROM levels WHERE (name LIKE '%CSP U7%');` ... I don't see anything with 2022. I see these
```
| CSP U7 20-21 Student Pilot Survey                    |
| CSP U7 20-21 Teacher Pilot Survey                    |
| CSP U7 Param Practice App_3_ExemplarIssue            |
| CSP U7L1 SFLP 20-21                                  |
| CSP U7L1 SFLP 20-21_2021                             |
| CSP U7L10 SFLP 20-21                                 |
| CSP U7L10 SFLP 20-21_2021                            |
| CSP U7L11 SFLP 20-21                                 |
| CSP U7L12 SFLP 20-21                                 |
| CSP U7L12 SFLP 20-21_2021                            |
| CSP U7L13 SFLP 20-21                                 |
| CSP U7L2 My Pet Rock Parameters                      |
| CSP U7L2 SFLP 20-21                                  |
| CSP U7L2 SFLP 20-21_2021                             |
| CSP U7L3 SFLP 20-21                                  |
| CSP U7L3 SFLP 20-21_2021                             |
| CSP U7L4 SFLP 20-21                                  |
| CSP U7L4 SFLP 20-21_2021                             |
| CSP U7L5 SFLP 20-21                                  |
| CSP U7L5 SFLP 20-21_2021                             |
| CSP U7L6 SFLP 20-21                                  |
| CSP U7L6 SFLP 20-21_2021                             |
| CSP U7L7 SFLP 20-21                                  |
| CSP U7L7 SFLP 20-21_2021                             |
| CSP U7L8 SFLP 20-21                                  |
| CSP U7L8 SFLP 20-21_2021                             |
| CSP U7L9 SFLP 20-21                                  |
| CSP U7L9 SFLP 20-21_2021                             |
| SP-VPL-21-mod3-CSP U7 Libraries Project Part 1_2021  |
| SP-VPL-21-mod3-CSP U7 Param Return Make RPS App_2021
```

Reply:
I found it here: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts/levels/U7%20Param%20Investigate%20Poetry%20App2022.level no "CSP" in the name, it seems

Reply:
When the cloning failed, we didn't remove any of the level files, so it makes sense that they already exist. I am surprised that no level files were created with a successful clone of this script, but I can't say for certain that that's not expected

Reply:
Ah thanks, everyone––makes sense